central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -646,3 +646,17 @@ func (ds *datastoreImpl) RemovePLOPsWithoutProcessIndicatorOrProcessInfo(ctx con
 
 	return int64(len(plopsToDelete)), nil
 }
+
+func (ds *datastoreImpl) RemovePLOPsWithoutPodUID(ctx context.Context) (int64, error) {
+	ds.mutex.Lock()
+	defer ds.mutex.Unlock()
+
+	deletedPlops, err := ds.storage.DeleteByQuery(ctx, search.NewQueryBuilder().
+		AddNullField(search.PodUID).ProtoQuery())
+
+	if err != nil {
+		return 0, nil
+	}
+
+	return int64(len(deletedPlops)), err
```

Comment:
How about wrapping error? Since we return it here we can remove 657-659 or explicitly return nil here and wrap above

Reply:
I got err and nil backwards here. I don't know how that happened.